src/app/carbon-estimation/carbon-estimation.component.spec.ts

```diff
@@ -47,16 +47,11 @@ describe('CarbonEstimationComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should set the chart height when the component is initialised', () => {
-    spyOnProperty(window, 'innerHeight').and.returnValue(1000);
-    spyOnProperty(window, 'innerWidth').and.returnValue(1000);
-    spyOnProperty(window.screen, 'height').and.returnValue(1080);
-    spyOnProperty(component.detailsPanel.nativeElement, 'clientHeight').and.returnValue(200);
```

Comment:
Now that the height isn't set in `ngOnInit` these spies would need to be set in the `beforeEach` section to take effect in time. Whilst it works for this test I think it might seem a bit odd to explicitly set the dimensions to desktop dimensions before every test since some are describing mobile behaviour. I've changed the test to just check that the height is set to a positive value initially.

Reply:
Yeah, seems reasonable.